static/src/javascripts/projects/commercial/modules/header-bidding/a9/a9.js

```diff
@@ -34,19 +34,24 @@ const bidderTimeout: number = 1500;
 
 const initialise = (): void => {
     onIabConsentNotification(state => {
-        // typeof state === 'boolean' means CCPA mode is on
-        const canRun =
-            typeof state === 'boolean'
-                ? !state
-                : state[1] && state[2] && state[3] && state[4] && state[5];
-
-        if (!initialised && canRun) {
-            initialised = true;
-            window.apstag.init({
+        if (!initialised) {
+            const apstagConfig = {
                 pubID: config.get('page.a9PublisherId'),
                 adServer: 'googletag',
                 bidTimeout: bidderTimeout,
-            });
+            }
+            // typeof state === 'boolean' means CCPA mode is on
+            if (typeof state === 'boolean') {
```

Comment:
Probably a more solid test to know if CCPA is in action is to test for the presence of window.__uspapi? I understand the typechecking way of what 'onIabConsentNotification' calls back with will be deprecated soon.